examples/rsp-next-ts/next.config.mjs

```diff
@@ -50,9 +51,10 @@ const withTM = require("next-transpile-modules")([
   "@spectrum-icons/workflow",
 ]);
 
-module.exports = withTM({
+let result = withTM({
   basePath:
     process.env.VERDACCIO && process.env.CIRCLE_SHA1
       ? `/reactspectrum/${process.env.CIRCLE_SHA1}/verdaccio/next`
       : "",
 });
+export default result;
```

Comment:
Was there a logic reason for this change or just cleaner code?

Reply:
it must be this way for esm